inference-engine/thirdparty/clDNN/src/network.cpp

```diff
@@ -464,6 +465,9 @@ void network_impl::execute(const std::vector<event::ptr>& events) {
     OV_ITT_SCOPED_TASK(itt::domains::CLDNN, "NetworkImpl::Execute");
     // Wait for previous execution completion
     reset_execution(false);
+    auto debug_config = debug_configuration::get_instance();
+    GPU_DEBUG_IF(debug_config->verbose >= 1)
+        std::cout << debug_config->prefix << "----------------------------------------------" << std::endl;
```

Comment:
I think it makes sense to unify such information prints. For example, using some helper function, which will be called at the beginning of each option. Something like this:
```
void debug_configuration::print_option_header(std::string option_name) {
    std::cout << debug_config->prefix << " -------- " << optioin_name << " -------- " << std::endl;
}
```

Reply:
Applied, thanks!

Reply:
Well, in that case I think we should add a macro like `#define GPU_DEBUG_PRINT std::cout << debug_config->prefix`

Reply:
@vladimir-paramuzov  Applied, thanks.